src/templates/about-page.js

```diff
@@ -13,26 +13,20 @@ export const AboutPageTemplate = ({
   const PageContent = contentComponent || Content;
 
   return (
-    <section className="section section--gradient">
-      <div className="container">
-        <div className="columns">
-          <div className="column is-10 is-offset-1">
-            <div className="section">
-              <h2 className="header-title title is-size-3 has-text-weight-bold is-bold-light">
-                {title}
-              </h2>
+    <section>
+      <div>
+        <div>
```

Comment:
Jeg tenker vi kan fjere <div /> også siden de heller ikke er i bruk lengre.

Reply:
Tenker det kanskje kan komme etterhvert som man jobber med filene? Denne branchen var mest for å få fjernet sass, slik at man ikke bygde mer på den.

Reply:
ok da tar vi det et annet sted